### PR TITLE
Move crds to chart

### LINKED ISSUE #1172: Remove automatic creation of CRD's from code
It forces the user to give the component permission to create CRD's which is something only the admin of the cluster should have.

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Move crds to chart

Fixes #1172 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by alvaroaleman** [comment]
Nice! Should we add a check to ensure the crds are established?

**Comment by mrIncompetent** [comment]
The controller wont start until they are as the API is not available until then afaik

File: config/kubermatic/templates/crd-groups.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: groups.kubermatic.k8s.io
```
Comment (alvaroaleman) [inline config/kubermatic/templates/crd-groups.yaml]: Where does this come from?

Comment (mrIncompetent) [reply config/kubermatic/templates/crd-groups.yaml]: Good question - @p0lyn0mial did you created this CRD during development?
Otherwise i'll remove it

Comment (p0lyn0mial) [reply config/kubermatic/templates/crd-groups.yaml]: no I did not. I think it is safe to remove this CRD.

**Comment by mrIncompetent** [comment]
Removed the groups CRD. @alvaroaleman PTAL

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**